settings.gradle.kts

```diff
@@ -9,7 +9,7 @@ dependencyResolutionManagement {
     repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
     repositories {
         maven {
-            setUrl("https://androidx.dev/snapshots/builds/10707469/artifacts/repository")
+            setUrl("https://androidx.dev/snapshots/builds/10836739/artifacts/repository")
```

Comment:
I didn't switch to `latest` since the package versions can change. eg. camera changed from 1.3.0 to 1.4.0 from previous version to latest.